src/models/role.model.ts

```diff
@@ -29,16 +33,14 @@ export class Role extends Entity {
   })
   id: string;
 
-  @belongsTo(() => User)
-  userId: string;
+  @hasMany(() => User)
```

Comment:
what type of relation are you trying to establish

Reply:
Here one Role can have many users right.

Example: One Role(Admin) can be assigned to many users.

Reply:
okay